src/index.tsx

```diff
@@ -12,14 +12,8 @@ ReactDOM.render(
   <Box style={styles.app}>
     <React.StrictMode>
       <Provider store={store}>
-        <Theme.Provider theme="default">
+        <Theme.Provider theme="twilio">
```

Comment:
twilio theme brings in twilio sans font without needing to manually set it anywhere.

Reply:
This makes me so happy, I need a better gif than the chef kiss one I always use